covid-19-support/src/components/Highlights.vue

```diff
@@ -1,51 +1,167 @@
 <template>
-    <div class="row highlights">
-        <div class="col-6 col-md-3 order-md-1">                    
-            <value-box :icon="valueBoxes[0].icon" :title="valueBoxes[0].title" :value="valueBoxes[0].value" class="bg-blue"/>
-        </div>
-        <div class="col-6 col-md-3 order-md-2">                    
-            <value-box :icon="valueBoxes[1].icon" :title="valueBoxes[1].title" :value="valueBoxes[1].value" class="bg-green"/>
-        </div>
-        <div class="col-6 col-md-3 order-md-4">                    
-            <value-box :icon="valueBoxes[2].icon" :title="valueBoxes[2].title" :value="valueBoxes[2].value" class="bg-green"/> 
-        </div>
-        <div class="col-6 col-md-3 order-md-3">              
-            <value-box :icon="valueBoxes[3].icon" :title="valueBoxes[3].title" :value="valueBoxes[3].value" class="bg-blue"/>
-        </div>
+<div class="row highlights">
+    <div 
+        class="col-6 col-md-3 order-md-1"
```

Comment:
This line unfortunately breaks the functionality on mobile, if you look above at the original code you will see the classes are different. There might be a nicer way to write what I did, I had wanted to use a v-for, but at the time I wasn't sure how best to handle those different classes, hence just wrote them out in full :)